politeiad/v1.go

```diff
@@ -0,0 +1,798 @@
+// Copyright (c) 2020-2021 The Decred developers
```

Comment:
Shouldn't we off all V1 stuff? This is an incompatible release with no hope of upgrading. I think we should just remove all V1 stuff everywhere.

Reply:
I'm all for it. We have to support it until cms migrates though.